### DISCUSSION TITLE
App Router

### DISCUSSION THREAD

**Discussion by Andrewx24** [boxyhq/saas-starter-kit#1623] [Ideas]
I see that this project has the page router. Will there be a app router used for this project since app router is a stable feature of Next.JS

**Comment by deepakprabhakara** [comment]
Hi @Andrewx24, we plan to update it to use the app router feature but there is no definitive timeline on it yet. I'll keep you posted. The upgrade will be done incrementally to reduce complexity.

**Reply by quacrobat** [reply]
hi @deepakprabhakara - any update on this ?
if not, how to benefit of app router in saas-starter-kit?
